Source/FieldSolver/SpectralSolver/SpectralSolverRZ.cpp

```diff
@@ -41,68 +43,97 @@ SpectralSolverRZ::SpectralSolverRZ (amrex::BoxArray const & realspace_ba,
+        algorithm = std::unique_ptr<PsatdAlgorithmRZ>(
+            new PsatdAlgorithmRZ(k_space, dm, n_rz_azimuthal_modes, norder_z, nodal, dt));
+    } else {
+        // Otherwise: use the Galilean algorithm
+        algorithm = std::unique_ptr<GalileanPsatdAlgorithmRZ>(
+            new GalileanPsatdAlgorithmRZ(k_space, dm, n_rz_azimuthal_modes, norder_z, nodal, v_galilean, dt));
+    }
 
     // - Initialize arrays for fields in spectral space + FFT plans
     field_data = SpectralFieldDataRZ(realspace_ba, k_space, dm,
                                      algorithm->getRequiredNumberOfFields(),
                                      n_rz_azimuthal_modes, lev);
 }
 
-
+/* \brief Transform the component `i_comp` of MultiFab `field_mf`
+ *  to spectral space, and store the corresponding result internally
+ *  (in the spectral field specified by `field_index`) */
 void
-SpectralSolverRZ::ForwardTransform (amrex::MultiFab const & field_mf,
-                                    int const field_index,
+SpectralSolverRZ::ForwardTransform (amrex::MultiFab const & field_mf, int const field_index,
                                     int const i_comp) {
     WARPX_PROFILE("SpectralSolverRZ::ForwardTransform");
     field_data.ForwardTransform(field_mf, field_index, i_comp);
-}
+};
 
+/* \brief Transform the two MultiFabs `field_mf1` and `field_mf2`
+ *  to spectral space, and store the corresponding results internally
+ *  (in the spectral field specified by `field_index1` and `field_index2`) */
 void
 SpectralSolverRZ::ForwardTransform (amrex::MultiFab const & field_mf1, int const field_index1,
                                     amrex::MultiFab const & field_mf2, int const field_index2) {
     WARPX_PROFILE("SpectralSolverRZ::ForwardTransform");
     field_data.ForwardTransform(field_mf1, field_index1,
                                 field_mf2, field_index2);
-}
+};
 
+/* \brief Transform spectral field specified by `field_index` back to
+ * real space, and store it in the component `i_comp` of `field_mf` */
 void
-SpectralSolverRZ::BackwardTransform (amrex::MultiFab& field_mf,
-                                     int const field_index,
+SpectralSolverRZ::BackwardTransform (amrex::MultiFab& field_mf, int const field_index,
                                      int const i_comp) {
     WARPX_PROFILE("SpectralSolverRZ::BackwardTransform");
     field_data.BackwardTransform(field_mf, field_index, i_comp);
-}
+};
 
+/* \brief Transform spectral fields specified by `field_index1` and `field_index2`
+ * back to real space, and store it in `field_mf1` and `field_mf2`*/
 void
 SpectralSolverRZ::BackwardTransform (amrex::MultiFab& field_mf1, int const field_index1,
                                      amrex::MultiFab& field_mf2, int const field_index2) {
     WARPX_PROFILE("SpectralSolverRZ::BackwardTransform");
     field_data.BackwardTransform(field_mf1, field_index1,
                                  field_mf2, field_index2);
-}
+};
```

Comment:
warning: extra ‘;’ [-Wpedantic]
```suggestion
}
```